contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -206,32 +178,29 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( eosio_global_state4, (continuous_rate)(inflation_pay_factor)(votepay_factor) )
    };
 
-   /**
-    * Defines `producer_info` structure to be stored in `producer_info` table, added after version 1.0
-    */
+   // Defines `producer_info` structure to be stored in `producer_info` table, added after version 1.0
    struct [[eosio::table, eosio::contract("eosio.system")]] producer_info {
-      name                  owner;
-      double                total_votes = 0;
-      eosio::public_key     producer_key; /// a packed public key object
-      bool                  is_active = true;
-      std::string           url;
-      uint32_t              unpaid_blocks = 0;
-      time_point            last_claim_time;
-      uint16_t              location = 0;
+      name                                                     owner;
+      double                                                   total_votes = 0;
+      eosio::public_key                                        producer_key; /// a packed public key object
+      bool                                                     is_active = true;
+      std::string                                              url;
+      uint32_t                                                 unpaid_blocks = 0;
+      time_point                                               last_claim_time;
+      uint16_t                                                 location = 0;
+      eosio::binary_extension<eosio::block_signing_authority>  producer_authority; // added in version 1.9.0
```

Comment:
This incorrectly brings in code changes from 1.9.0. There should only be documentation changes in this PR.

Reply:
corrected